README.md

```diff
@@ -115,6 +115,48 @@ jobs:
           sceptre_subcommand: 'launch -y dev'
 ```
 
+```yaml
+# Function: Run pre-commit checks and deploy 'dev' stack group if checks pass
+# Trigger: Push/Merge to all branches, but deployment reserved to 'main' branch
+# Plugins: Install from a Pipfile using the specified version of Pipenv 
```

Comment:
Instead of making this action more complex i'm wondering if it would be better to have users run separate steps for the specific action before running the sceptre action?  There are already a few pipenv[1] and requirements[2] actions in the GH action marketplace.

[1] https://github.com/marketplace?type=actions&query=pipenv+
[2]  https://github.com/marketplace?type=actions&query=python+requirements+

Reply:
Unfortunately, this action runs within a container, so dependencies installed in previous CI steps aren't available to the Sceptre action. We could drop the container, but then this action is barely more than just using:

```yaml
- name: Deploy common configuration to develop account
  run: sceptre launch prod --yes
```

I agree that this might be adding too much complexity to the action, so I'm open to closing this PR.

Reply:
you are correct @BrunoGrandePhD, i forgot that GH actions are scoped to containers.